src/Tabs/Tabs.js

```diff
@@ -1,111 +1,129 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import { BrowserRouter, Link } from 'react-router-dom';
 import React, { Component } from 'react';
 
-export const Tabs = props => {
-    const { children, className, ...rest } = props;
+export const Tab = props => {
+    const { children, className, content, disabled, id, onClick, selectedId, tabContentProps, tabLinkProps, url, ...rest } = props;
```

Comment:
It looks like there are a number of these props that are not defined in the `propTypes` or `propDescriptions`.  The `Tab` component is only showing `id`, `children`, `className` and `url`.